packages/components/src/Layout/Semantics/Story/ExamplePage.story.tsx

```diff
@@ -345,3 +345,50 @@ export const ScrollAllAreas = () => (
     </Page>
   </Highlighter>
 )
+
+export const AsideCollapse = () => (
+  <Highlighter>
+    <Page>
+      <Layout hasAside>
+        <Aside collapse scrollWithin p="large" width="200px">
+          I'm baby man braid cold-pressed seitan sartorial, tumblr ennui selfies
+          chia twee subway tile af 90's celiac. Gochujang distillery tumeric
+          flannel lumbersexual gastropub fashion axe viral neutra. Pickled
+          narwhal everyday carry activated charcoal succulents occupy YOLO
+          yuccie forage seitan kitsch. Celiac bespoke cloud bread asymmetrical
+          bicycle rights XOXO cold-pressed hashtag listicle hell of migas.
+          Chillwave brooklyn fam occupy microdosing leggings. Poke af cornhole
+          hot chicken. Portland tattooed +1 chia ennui.
+        </Aside>
+        <Section scrollWithin p="xxlarge" as="main">
+          <Heading>Page title</Heading>
+          <Tabs>
+            <TabList distribute>
+              <Tab>Blue</Tab>
+              <Tab>Green</Tab>
+              <Tab>Purple</Tab>
+              <Tab>Yellow</Tab>
+              <Tab>Orange</Tab>
+              <Tab>Red</Tab>
+              <Tab>White</Tab>
+              <Tab>Black</Tab>
+            </TabList>
+            <TabPanels>
+              <TabPanel>{longContent}</TabPanel>
+            </TabPanels>
+          </Tabs>
+        </Section>
+        <AsideAlt scrollWithin p="xxlarge" width="10rem">
+          I'm baby man braid cold-pressed seitan sartorial, tumblr ennui selfies
+          chia twee subway tile af 90's celiac. Gochujang distillery tumeric
+          flannel lumbersexual gastropub fashion axe viral neutra. Pickled
+          narwhal everyday carry activated charcoal succulents occupy YOLO
+          yuccie forage seitan kitsch. Celiac bespoke cloud bread asymmetrical
+          bicycle rights XOXO cold-pressed hashtag listicle hell of migas.
+          Chillwave brooklyn fam occupy microdosing leggings. Poke af cornhole
+          hot chicken. Portland tattooed +1 chia ennui.
+        </AsideAlt>
```

Comment:
Can we use a simpler example here so it's easier to "see" what's being tested here:

```suggestion
        <Aside collapse>Aside</Aside>
        <Section as="main">Section</Section>
```